python/lsst/afw/image/exposure.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import
+
+from ._exposure import *
+
+def __reduce__(self):
+    from lsst.afw.fits import reduceToFits
+    return reduceToFits(self)
+
+ExposureI.__reduce__ = __reduce__
+ExposureF.__reduce__ = __reduce__
+ExposureD.__reduce__ = __reduce__
+ExposureU.__reduce__ = __reduce__
+#ExposureL.__reduce__ = __reduce__
```

Comment:
Not my code, but could this be related to [RFC-227](https://jira.lsstcorp.org/browse/RFC-227)?